Formula/python@3.9.rb

```diff
@@ -83,6 +83,13 @@ class PythonAT39 < Formula
       url "https://raw.githubusercontent.com/Homebrew/formula-patches/33a9d63f/python/arm64-3.9.patch"
       sha256 "167e328cf68e9ec142f483fda9fafbb903be9a47dee2826614fdc24b2fbe6e06"
     end
+
+    # Further patch for Big Sur, remove in 3.9.2
+    # https://github.com/python/cpython/pull/23556
+    patch do
+      url "https://github.com/fxcoudert/cpython/commit/cd26ccbb.patch?full_index=1"
+      sha256 "59c98f991f839d610d53ca5c4af1464a99adf6c85e807f8a676b3e41c5dbe0a2"
+    end
```

Comment:
Can we move this to formula-patches.

Reply:
It's the commit from the upstream pull request (which I opened myself). I tried to use the PR URL, but then it complains.

Reply:
You can use the commit hash in the PR vs from your branch: https://github.com/python/cpython/commit/cd26ccbbf97be0d729196df3ec2a25ef4f0c57f8.patch?full_index=1 . 

Disallowing the PR URL itself was removed because the contents of the PR can change. If there are multiple commits, then using formula-patches can make more sense.